### PR TITLE
Single sig options

### PR DESCRIPTION
**Pull request by jgriffiths**
All Electrum Liquid sessions to available options, and allow overriding the Electrum server to connect to.

### CODE REVIEW

File: green_cli/green.py [hunk 1]
```diff
@@ -40,6 +40,8 @@ def _get_authenticator(options):
 @click.option('--auth-config', default=None, help='Additional json config passed to the authenticator')
 @click.option('--config-dir', '-C', default=None, help='Override config directory.')
 @click.option('--compact', '-c', is_flag=True, help='Compact json output (no pretty printing)')
+@click.option('--electrum-url', default=None, type=str, help='Use the given Electrum server (single-sig only)')
+@click.option('--electrum-tls', default=None, type=bool, help='Connect to electrum using TLS (single-sig only)')
```
Comment (LeoComandini) [inline green_cli/green.py:44]: nit: actually this params are used for multi-sig SPV as well

Comment (LeoComandini) [reply green_cli/green.py:44]: IIRC these params should be passed at `connect`, thus should be added to `session_params`

File: green_cli/liquid.py [hunk 2]
```diff
@@ -27,7 +29,7 @@
 
 # Restrict networks to liquid networks and default to localtest-liquid
 params = {p.name: p for p in green.params}
-params['network'].type = click.Choice(['liquid', 'testnet-liquid', 'localtest-liquid'])
+params['network'].type = click.Choice(reversed([n for n in gdk.get_networks() if 'liquid' in n]))
```
Comment (LeoComandini) [inline green_cli/liquid.py:32]: Do we want to add single-sig Liquid Mainnet as well?

**Comment by LeoComandini** [comment]
utACK b354bd7af2cc0e0c6bf8861b3698dfaed8269a8e, with comments

File: green_cli/green.py (hunk 1 above)
Comment (jgriffiths) [reply green_cli/green.py]: I've updated the text description (forgot about SPV).

The params are already added to `session_params` via `optional_keys`.

File: green_cli/liquid.py (hunk 2 above)
Comment (jgriffiths) [reply green_cli/liquid.py]: Good point, I've disabled it until our internal acceptance testing is completed.

**Pull request merged**